app/main.ts

```diff
@@ -1,7 +1,15 @@
 import { bootstrap }    from '@angular/platform-browser-dynamic';
-
+import { HTTP_PROVIDERS } from "@angular/http";
 import { AppComponent } from './app.component';
-
 import { appRouterProviders } from "./app.routes";
 
-bootstrap(AppComponent,[appRouterProviders]);
+
```

Comment:
This is the way  for register a router provider in the bootstrap method,  Angular provide   the Component Router service for present a particular component from a given URL, a router configuration is needed and that come from "app.routes" exports